test/tool-option-parsing.py

```diff
@@ -279,10 +280,13 @@ def test_interactive_wayland(self):
 
 if __name__ == '__main__':
     with tempfile.TemporaryDirectory() as tmpdir:
+        # Use our own test xkeyboard-config copy.
+        os.environ['XKB_CONFIG_ROOT'] = top_srcdir + '/test/data'
         # libxkbcommon has fallbacks when XDG_CONFIG_HOME isn't set so we need
         # to override it with a known (empty) directory. Otherwise our test
         # behavior depends on the system the test is run on.
         os.environ['XDG_CONFIG_HOME'] = tmpdir
+        del os.environ['HOME']
```

Comment:
we shouldn't need this one right? If we do, then something is likely off with the test.

edit, nevermind, I think that catches the `$HOME/.xkb/` fallback for the merged kcgst files.

Reply:
Added a comment to clarify.